ports/stm32/mpbthciport.c

```diff
@@ -137,29 +132,18 @@ int mp_bluetooth_hci_uart_write(const uint8_t *buf, size_t len) {
     return 0;
 }
 
-// Callback to copy data into local hci_uart_rx_buf_data buffer for subsequent use.
-STATIC int mp_bluetooth_hci_uart_msg_cb(void *env, const uint8_t *buf, size_t len) {
+// Callback to forward data from rfcore to the bluetooth hci handler.
+STATIC void mp_bluetooth_hci_uart_msg_cb(void *env, const uint8_t *buf, size_t len) {
     (void)env;
```

Comment:
No longer needed.

Reply:
Does everything always pass a `mp_bluetooth_hci_uart_readchar_t` now? Does this need to be a void* anymore or can we just fix the signature everywhere?